### ISSUE TITLE
Consider adding a link to the Matrix community space in the socials section of the footer

### ISSUE THREAD

**Issue by Gnuxie** [matrix-org/matrix.org#2354]
Related: https://github.com/matrix-org/matrix.org/issues/2353
Context: https://matrix.to/#/%23matrix.org-website%3Amatrix.org/%24BH-D9erUnfK32zeO87bIM2-s4qHTH8XZlpUzIAy6ehg?via=matrix.org&via=element.io&via=envs.net

**Comment by HarHarLinks** [comment]
I agree #community:matrix.org or perhaps the #foundation-office:matrix.org would make sense
